requirements/edx/base.txt

```diff
@@ -162,9 +162,9 @@ mongoengine==0.10.0       # via -r requirements/edx/base.in
 more-itertools==8.3.0     # via -r requirements/edx/paver.txt, zipp
 mpmath==1.1.0             # via sympy
 mysqlclient==1.4.6        # via -r requirements/edx/base.in
-newrelic==5.12.1.141      # via -r requirements/edx/base.in, edx-django-utils
+newrelic==5.14.0.142      # via -r requirements/edx/base.in, edx-django-utils
```

Comment:
@robrap It looks like this release has some new stuff involving custom attributes; does any of it look useful or concerning to you? https://docs.newrelic.com/docs/release-notes/agent-release-notes/python-release-notes